integration-tests/jta/src/main/java/org/apache/camel/quarkus/component/jta/it/JtaRoutes.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.jta.it;
+
+import org.apache.camel.builder.RouteBuilder;
+
+public class JtaRoutes extends RouteBuilder {
+    @Override
+    public void configure() throws Exception {
+        from("direct:test")
+                .transacted().transform().constant("Hello World");
```

Comment:
Do not see any coverage for the individual various TransactionPolicies. Would it be hard to add a test route per policy?

Reply:
Sure, I will add more tests for all the JTA transaction policies. Thanks for pointing it out !